ui/src/organizations/components/Members.tsx

```diff
@@ -34,17 +30,29 @@ import {
   memberAddFailed,
 } from 'src/shared/copy/v2/notifications'
 
+export interface UserObject {
+  [key: string]: {
```

Comment:
```suggestion
  [userID: string]: User
```

Is the value here just a User interface or maybe `Pick<User, 'id' | 'name' | 'links'>`

Reply:
Yes it's the user interface. I gave it the same type